source/api/discovery/0.1/index.md

```diff
@@ -0,0 +1,608 @@
+---
+title: "IIIF Change Discovery API 0.1"
+title_override: "IIIF Change Discovery API 0.1"
+id: discovery-api
+layout: spec
+cssversion: 3
+tags: [specifications, presentation-api]
+major: 0
+minor: 1
+patch: 0
+pre: final
+redirect_from:
+  - /api/discovery/index.html
+  - /api/discovery/0/index.html
+---
+
+## Status of this Document
+{:.no_toc}
+__This Version:__ {{ page.major }}.{{ page.minor }}.{{ page.patch }}{% if page.pre != 'final' %}-{{ page.pre }}{% endif %}
+
+__Latest Stable Version:__ None
+
+__Previous Version:__ None
+
+**Editors**
+
+  * **[Michael Appleby](https://orcid.org/0000-0002-1266-298X)** [![ORCID iD]({{ site.url }}{{ site.baseurl }}/img/orcid_16x16.png)](https://orcid.org/0000-0002-1266-298X), [_Yale University_](http://www.yale.edu/)
+  * **[Tom Crane](https://orcid.org/0000-0003-1881-243X)** [![ORCID iD]({{ site.url }}{{ site.baseurl }}/img/orcid_16x16.png)](https://orcid.org/0000-0003-1881-243X), [_Digirati_](http://digirati.com/)
+  * **[Robert Sanderson](https://orcid.org/0000-0003-4441-6852)** [![ORCID iD]({{ site.url }}{{ site.baseurl }}/img/orcid_16x16.png)](https://orcid.org/0000-0003-4441-6852), [_J. Paul Getty Trust_](http://www.getty.edu/)
+  * **[Jon Stroop](https://orcid.org/0000-0002-0367-1243)** [![ORCID iD]({{ site.url }}{{ site.baseurl }}/img/orcid_16x16.png)](https://orcid.org/0000-0002-0367-1243), [_Princeton University Library_](https://library.princeton.edu/)
```

Comment:
I'm happy to be associated with this, but TBH I've done very little for this initiative over the past year and I'm not sure it's fair to the rest of the group. Will let others decide.